File: Programs/Gradebook/Python/Student.py
class Student(object):
    def __init__(self, id, name, gender):
        self.id = id
        self.name = name
        self.gender = gender

    def __repr__(self):
        return repr((self.id, self.name, self.gender))

    def getId(self):
        return self.id

    def getName(self):
        return self.name

    def getGender(self):
        return self.gender

    def setId(self, id):
        self.id = id

    def setName(self, name):
        self.name = name

    def setGender(self, gender):
        self.gender = gender

File: Programs/Gradebook/Scala/Student.scala
class Student(val newId: Int, val newName: String, val newGender: String){
	var id: Int = newId
	var name: String = newName
	var gender: String = newGender

	def getId(): Int = {
		return id
	}

	def getName(): String = {
		return name
	}

	def getGender(): String = {
		return gender
	}

	def setId(newId: Int) {
		id = newId
	}

	def setName(newName: String) {
		name = newName
	}

	def setGender(newGender: String) {
		gender = newGender
	}
}